src/drivers/sf1xx/sf1xx.cpp

```diff
@@ -594,24 +593,17 @@ SF10A::cycle()
 		return;
 	}
 
-
-	/* Trigger measurement */
-	if (OK != measure()) {
-		DEVICE_DEBUG("measure error lidar");
-	}
-
```

Comment:
No, it not required according to datasheet. You write '\0' byte once to start measurements and then only poll for updates.

```
Values in the SF11 are held at specific locations and once a value has been read or written the location remains active until changed.
This is useful to simplify the code needed for the continuous monitoring of a value. For example, to update the distance
continuously, set the distance location once then read the value in a loop like the Arduino code below:
```
![ds](https://cloud.githubusercontent.com/assets/503830/17240131/6522160a-557a-11e6-8003-c771c67d5069.png)